src/frontend/src/lib/components/core/Loader.svelte

```diff
@@ -128,7 +128,7 @@
 		<div in:fade={{ delay: 0, duration: 250 }}>
 			<Modal testId={LOADER_MODAL}>
 				<div class="stretch">
-					<ImgBanner width="100%" src={banner} size="small" />
+					<ImgBanner styleClass="" src={banner} />
```

Comment:
since `styleClass` defaults to `undefined`, no need to pass it empty here, right?

Reply:
you are right